app/assets/stylesheets/modules/projects.css.scss

```diff
@@ -518,4 +523,13 @@ $card-background: rgba(235,235,235,1);
     margin-top: 0;
     margin-bottom: 0;
   }
+
+  p.empty-copy {
+    display: none;
+    font-size: 1.3em;
+    font-style: italic;
+    color: rgba(0,0,0,0.5);
```

Comment:
And since we're making some other changes, we can update this color to use a consistent color with other light greys on this same page... `color: darken($light-grey, 20%);`